src/test/java/com/rtbhouse/utils/avro/FastSpecificDeserializerGeneratorTest.java

```diff
@@ -508,6 +515,50 @@ public void shouldReadMapOfRecords() {
         Assert.assertEquals("abc", map.get("2").get("testStringUnion"));
     }
 
+    @Test
+    public void testNullElementMap() {
```

Comment:
Please rename the test method to `shouldDeserializeNullElementInMap`.

Reply:
Also please consider adding similar test methods to both `FastGenericDeserializerGeneratorTest` and `FastGenericSerializerGeneratorTest`. This will improve coverage in case of distinct implementations for specific and generic records serde.